src/functions-and-arrays.js

```diff
@@ -1,19 +1,81 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(number1, number2){ 
+  let max = 0;
+  if(number1 > number2){
+    max = number1;
+  } else if (number2 > number1){
+    max = number2;
+  } else {
+    max = number1; 
+  }
+  return max;
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(words){
+  if(words.length === 0){
+    return null;
+  }
+  
+  if(words.length ===1){
+    return words [0]
+  } 
```

Comment:
```suggestion
  if(words.length === 1){
    return words[0]
  } 
```
Pijadas pero que el TOC no me permite pasar jejeje